### ISSUE TITLE
Double English existential construction?

### ISSUE THREAD

**Issue by amir-zeldes** [LeonieWeissweiler/UCxn#7]
Hi all - since we said we'd try discussing guidelines on the repo, here's a question for everyone: for English we defined several different existential constructions, including expletive "there (is)" and use of a strong existence verb like "some doubt exists".

I now have a sentence like this:

  * Suppose there exists a path between y1 and y3

What do we think is the right thing to do here? Should this sentence have:

```CoNLL-U
3	exists	exist	VERB	VBZ	Mood=Ind|Person=3|Tense=Pres|VerbForm=Fin	1	ccomp	1:ccomp	Cxn=Existential-CopPred-ThereExpl,Existential-ExistPred-NoExpl
```

If so, it makes the second label with "NoExpl" a little awkward... Any thoughts?

**Comment by nschneid** [comment]
Looks like a hybrid strategy, maybe `Cxn=Existential-ExistPred-ThereExpl`

**Comment by amir-zeldes** [comment]
Hm, it's easy enough to just define another label, but I'd be a little unhappy to have an ultra-rare one that appears only in this one specific sentence... I was initially also unhappy with having two `Existential` annotations on the same node, but thinking about it some more, maybe it's actually in the spirit of CxG to just say that this instantiates both constructions, so they should both be annotated? I mean, the fronted "there" construction has an existential signification and the verb "exist" also realizes a strategy to express existence, so maybe having two annotations is correct?

**Comment by nschneid** [comment]
Actually I just checked and `Existential-ExistPred-ThereExpl` is already in the [docs](https://github.com/LeonieWeissweiler/UCxn/blob/main/docs/UCxn-v1.pdf).

If we say constructions are defined by meaning, I don't want to have 2 annotations giving the impression that there are two separate assertions of existence. It would likely be translated with a non-hybrid strategy, right?

**Comment by amir-zeldes** [comment]
Yeah, it's definitely not two assertions of existence. But it is possible for two constructions to 'conspire' together to express the same meaning, and the listener can interpret it as one instance of an event. 

For example if we say "It was so big that it fell over as a result", there are two signifiers of resulthood: the lexical connective PP "as a result" and the syntactic causal excess construction. From a CxG perspective I think both of these are constructions (one is lexically specified, but that doesn't matter), and both have a 'result' meaning, but using them concurrently doesn't mean that there are two results. Would we want to create a special hybrid type just to describe this conjunction of two simultaneous constructions? If we go this route, I think we would end up with a very large inventory of labels.

**Comment by nschneid** [comment]
> For example if we say "It was so big that it fell over as a result", there are two signifiers of resulthood: the lexical connective PP "as a result" and the syntactic causal excess construction.

Yes, it's possible for language to be redundant—but there "as a result" is an adjunct that can be omitted without affecting the meaning. "There exist" doesn't feel redundant in quite the same way, though indeed there are alternatives with just "there" or just "exist".

With "there exist", neither word can simply be omitted. You could view "exist" as providing syntactic support (a bleached verb) in the there-cxn, or "there" as providing syntactic support (a nonreferential subject) in the exist-cxn. Is it right to say that both are happening simultaneously? I don't know. If we were trying to provide a full story of constructional compositionality, we would have to formalize that.

For me, intuitively, a speaker who says "there exists a path" is not overspecifying the existential relationship; they are just adopting a strategy that happens to use two elements of form that do not always co-occur. (Incidentally, "there exist" is quite idiomatic in mathematical jargon.)

**Comment by amir-zeldes** [comment]
Thanks for those comments, this is interesting! I think ultimately there are questions here that touch on the cognitive aspects of CxG. I think it's clear that "there exists" by itself could be a unique construction because construction grammar allows for that; but I also think UCxn represents the kind of CxG view where constructions are licensed by being unpredictable from the sum of their parts, and I'm not sure there is anything here beyond a combination of expletive "there" and the normal verb "exist". 

To use a different example, even if the string "hang up the phone" is very common and probably entrenched, we wouldn't want to treat it as a special construction - rather, it is the combination of a lexical phrasal verb construction "hang up" meaning "stop talking on a device", and "the phone" is one possible NP that can appear as an object (like "the intercom", "that office phone", "your cell" or a variety of other phone-like object NPs). Similarly, the tense can vary to "hung up the phone" and we wouldn't call it a hybrid - it's just an application of the past tense construction + lexical construction "hang up" (which has a strong past based on the link to 'hang') + a standard NP construction.

If that's more or less how others understand CxG, then I think we can reformulate the question like this: is there anything special about "there exists", or is it just another verb that's compatible with the `ThereExpl` construction? In either case, does the existential meaning of "exist" differ from that in `ExistPred-NoExpl`? If it's just a combination of the two, I think the compositionality-based reasoning above makes "two different strategies instantiated simultaneously" more compelling. If we are saying it should get a unique label, I would like to understand how it differs from the sum of its parts. Of course, there are also views of CxG where anything entrenched, no matter how compositional, is a construction, but if someone wants to argue UCxn should adopt this view then maybe we should have that discussion first.
